src/event_model/schemas/event_descriptor.json

```diff
@@ -10,22 +10,47 @@
                 "data": {
                     "title": "Data",
                     "description": "The actual measurement data",
-                    "type": "object"
+                    "anyOf": [
+                        {
+                            "type": "object"
+                        },
+                        {
+                            "type": "null"
+                        }
+                    ],
+                    "default": null
```

Comment:
Hmm, this is actually saying that downstream clients now need to cope with the presence of this key, and its value being nul...